Documentation/helm-operator.md

```diff
@@ -12,24 +12,11 @@ Installs [rook](https://github.com/rook/rook) to create, configure, and manage C
 
 This chart bootstraps a [rook-ceph-operator](https://github.com/rook/rook) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-## Prerequisites
-
-* Kubernetes 1.11+
-
-### RBAC
-
-If role-based access control (RBAC) is enabled in your cluster, you may need to give Tiller (the server-side component of Helm) additional permissions. **If RBAC is not enabled, be sure to set `rbacEnable` to `false` when installing the chart.**
-
-```console
-# Create a ServiceAccount for Tiller in the `kube-system` namespace
-kubectl --namespace kube-system create sa tiller
+> Note: *Helm `v2.x`* is deprecated version, please upgrade to **Helm `v3.x`** ASAP.
 
-# Create a ClusterRoleBinding for Tiller
-kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+## Prerequisites
 
-# Patch Tiller's Deployment to use the new ServiceAccount
-kubectl --namespace kube-system patch deploy/tiller-deploy -p '{"spec": {"template": {"spec": {"serviceAccountName": "tiller"}}}}'
-```
+* Kubernetes 1.17+
```

Comment:
I see that the official docs for helm say that only the last three versions of K8s are supported, which puts it at 1.17. However, we have had much better luck supporting Rook than that limit. We run our helm integration tests all the way from K8s 1.11 to 1.19 and they have been passing. Let's rely on those tests for our min version and see if we can keep it the same as Rook's min version. There is no need to restrict it to 1.17 or newer if the Rook chart is tested and verified working on older K8s versions.

Reply:
Are the integration tests also installing an older version of Helm to match the older version of Kubernetes?

It might be worth noting here the minimum version of Helm as a prerequisite. Then using the Helm support matrix to give the supported version of Kubernetes (https://helm.sh/docs/topics/version_skew/).

Reply:
I believe the tests are just installing the same version of helm and it's working everywhere. Sounds good to link to the helm docs so they can determine for themselves if their version of helm is expected to be compatible, but so far we haven't had an issue with this.

Reply:
Should we be attempting to test with the same support matrix as Helm? ie, for integration tests against K8S 1.13, we use Helm 3.0.x.

Also noting that Helm 3 support only goes back as far as 1.13, so stating K8S 1.11 as the min version may not be a good idea.

Reply:
Helm 2 is a deprecated version. How about set k8s 1.13 as the min version?

Reply:
Yes, that works thanks